starterkits/drupal/api/preview/route.ts

```diff
@@ -0,0 +1,45 @@
+import { draftMode } from 'next/headers';
+import { redirect } from 'next/navigation';
+import { NextRequest, NextResponse } from 'next/server';
+import drupal from '../../../../util/drupal/drupal';
```

Comment:
After running `ddev setup-drupal` this seems to be the correct path:

```suggestion
import drupal from '../../../util/drupal/drupal';
```

Reply:
Or better:

```

import drupal from '@/util/drupal/drupal';
```

Reply:
Hmm, I thought I had fixed that but apparently not; will take a look. But yes, agreed, that should be `@/util/drupal/drupal`